### ISSUE TITLE
Plot is not displayed in line

### ISSUE THREAD

**Issue by hkiel** [OpenModelica/jupyter-openmodelica#2]
I successfully installed jupyter-notebook and the openmodelica kernel on mac (using Python2.7).
Simulation works and even plotting (with `plot(x);` a window with OMPlot opens, with `plot(x)` (without semicolon) the plot was shown in line.
However, since I installed anew (tried if Python3 works, but did not, so I reinstalled everything for 2.7) plotting no longer works in line ("No result File Generated"), external `plot(x);` works fine.
Anyone got an idea what is going on?

**Comment by sjoelund** [comment]
I believe omniORB (and thus OMPython) does not work in Python 3. You need to have omniidl including the Python plugin in order to to use OMPython (and the jupyter notebook); perhaps something went wrong there. Or does everything except the plotting work?

**Comment by hkiel** [comment]
Everything works except plotting in the notebook. Opening external plot with `plot(x);` does show correct output, but ONLY as a window.
I use Python 2.7 because Python3 does not work.

**Comment by hkiel** [comment]
@sjoelund I may have a clue why it does not work:
If plot() is used with semicolon, it is not seen as a plotcommand (expected to end with ')'), so OMPython opens OMPlot when using "plot(x);"

One step earlier, the simulate() command is triggered by

```
               try:
                   val=self.omc.sendExpression(code)
                   try:
                      self.matfile=val['resultFile']
                   except:
                      pass
```

where val is a String (I see val[0]='r', val[1]='e', val[2]='c',... -> "record Simulationresult...")
I guess `val['resultFile']` just fails and therefore `self.matfile` is never set (which I can confirm).

Any solution??? How can `val` become a named record?

**Comment by hkiel** [comment]
fixed in OpenModelica/OMPython#16
